analysed-packages/ca-certificates-debian/version-20211016/ca-certificates-debian-20211016-SPDX2TV.spdx

```diff
@@ -0,0 +1,2046 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_ca-certificates-debian-20211016.tar.gz_1719409425.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Parian Golchin
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of ca-certificates-debian 20211016.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-06-26T13:43:46Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: ca-certificates-debian-20211016.tar.gz
```

Comment:
PackageName has suffix tar.gz

Reply:
In general: Why our package has this "debian" in the name? This of O.Fendt does not.

Reply:
PackageName has always suffix tar.gz  or tar.xz based on what was the original upload format.

https://github.com/Open-Source-Compliance/package-analysis/blob/main/analysed-packages/ca-certificates/version-20230311/ca-certificates-20230311-SPDX2TV.spdx#L31

"In general: Why our package has this "debian" in the name? This of O.Fendt does not." -> That is a very good question. To be honest, I had a doubt to add debian or not. Because as you mentioned he did not consider debian but for other packets that they were debian, he did that  (see here https://github.com/Open-Source-Compliance/package-analysis/tree/main/analysed-packages/adduser-debian/version-3.137)

Reply:
@dierberg Debian Suffix removed https://github.com/DFE/package-analysis/pull/13/commits/a1a595d8b011a312d6091edd80c414eb9f703e18
Thank you for your review! :)